sigs/addons/MIGRATION_TO_CORE.md

```diff
@@ -0,0 +1,32 @@
+# Migration From TF-Addons To TensorFlow Core
+
+### In-Progress & Previous Migrations:
+https://github.com/tensorflow/addons/projects/2/
+
+### Process 
+1. Create an issue in TensorFlow Addons for a candidate that you think should be 
+migrated. 
+2. The SIG will evaluate the request and add it to the `Potential Candidates` section 
+of our GitHub project.
+3. If it's agreed that a migration makes sense, an RFC needs to be written to discuss 
+the move with a larger community audience. 
+    * If the transition will impact tf-core and Keras then submit the RFC to 
+    [TensorFlow Community](https://github.com/tensorflow/community)
+    * Additions which only subclass Keras APIs should submit their migration proposals to 
+    [Keras Governance](https://github.com/keras-team/governance)
+    
+4. A sponsor from the TF/Keras team must agree to shepard the transition.
+   * If no sponsor is obtained after 45 days the RFC will be rejected and will remain 
```

Comment:
I am ok to put a number here but I think that we need a realistic evaluation by the TF team. Is 45 days something realistic for the internal process?

Reply:
I think we could align to 1  month. See https://github.com/tensorflow/community/blob/master/governance/TF-RFCs.md#rfc-process

Reply:
Hmmm, would like to align here but 45 days ensures we overlap with a TFA monthly meeting and lets us get help from TF internal team members to help find sponsors. @theadactyl @ewilderj is there anyway we could make 45 days the standard amount of time for RFC sponsorship? I imagine this would be useful for multiple SIGs that have monthly meetings.

Reply:
Yes it is the same if we could extend the standard to 45 days is ok